Xamarin.Forms.Platform.WP8/Forms.cs

```diff
@@ -91,14 +92,27 @@ public WP8DeviceInfo()
 
 			public override Size ScaledScreenSize { get; }
 
-			public override double ScalingFactor
+			public override double ScalingFactor => _scalingFactor;
+
+			public override void BeginDeviceOrientationNotifications()
+			{
+				throw new NotImplementedException();
+			}
+
+			public override void EndDeviceOrientationNotifications()
 			{
-				get { return _scalingFactor; }
+				throw new NotImplementedException();
```

Comment:
Is WP8 still in active development? I'm not sure if it has a sensor manager. For now, device orientation changes are not supported.